explorer_frontend/src/features/account-connector/components/MainScreen.tsx

```diff
@@ -156,9 +159,13 @@ const MainScreen = () => {
               overrides={{
                 Root: {
                   style: {
-                    flex: 1, // Makes input take full available width
+                    flex: 1,
                     height: "48px",
                     background: COLORS.gray700,
+                    paddingLeft: "12px", // Add left padding
+                    paddingRight: "12px", // Add right padding
+                    maxWidth: "80%", // Ensure it doesn't stretch too far
+                    borderRadius: "8px", // Soften the edges for better UI
```

Comment:
No, this is a standard input from ui-kit and we should not modify things like border radius. It's unified across all the app and the website.